website/source/api/client.html.md

```diff
@@ -579,6 +618,13 @@ The table below shows this endpoint's support for
 | ---------------- | ------------ |
 | `NO`             | `node:write` |
 
+### Parameters
+
+- `node_id` `(string: <optional>)` - Specifies the node to target. This is
+  required when the endpoint is being access via a server. This is specified as
```

Comment:
What does it mean to say "this is specified as part of the URL"?

being access -> being accessed